Packages/com.chisel.core/Chisel/Core/API.private/Native/CSGTree.Native.cs

```diff
@@ -209,21 +210,20 @@ private static GeneratedMeshContents GetGeneratedMesh(int treeNodeID, GeneratedM
             { 
                 var ignoreNodeIDsPtr	= (ignoreNodes != null) ? ignoreNodeIDsHandle.AddrOfPinnedObject() : IntPtr.Zero;
 
-                // TODO: remove this ...
-                Matrix4x4 treeLocalToWorldMatrix = Matrix4x4.identity;
+                var workAround = Matrix4x4.identity;
```

Comment:
is this used?

Reply:
It's used in native side, but will be removed when we switch to a managed solution